src/content/docs/apis/nerdgraph/tutorials/provisions-your-subscriptions-nerdgraph.mdx

```diff
@@ -3,39 +3,34 @@ title: "NerdGraph tutorial: Partner and reseller subscriptions"
 
-## Get started: endpoint and API key [#endpoint]
+NerdGraph's provisioning functionality and the [Partnership API](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partner-api) are compatible when it comes to the products supported by both (APM, Mobile, Insights, Browser, Synthetics, and Infrastructure). If you use NerdGraph to create a subscription for any of the newer products, you'll no longer be able to update the account using the Partnership API.
 
-The provisioning API uses your [user key](/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key).
 
-The endpoint for the Provisioning API is:
+## Get started: endpoint and API key [#endpoint]
 
-```
-https://api.newrelic.com/graphql
-```
+For NerdGraph basics, see [Introduction to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#explorer).
 
-To get started, run the following command:
+The NerdGraph provisioning functionality uses a mutation called `organizationProvisioningUpdatePartnerSubscription`. Here's an example request making subscription changes to a specific account: 
 
 ```
 curl -v -d'{"query": "mutation { organizationProvisioningUpdatePartnerSubscription(accountId: <var>PARTNER_ACCOUNT_ID</var>, affectedAccountId: <var>ACCOUNT_ID</var>, 
-products: [{id: 9200, name: \"Traces\", unitsOfMeasure: [{unit: SPANS_IN_MILLIONS, quantity: 40}]}]) {enqueued errors {message path } } }"}' -H'Content-type: application/json' -H'Api-key: <var>PERSONAL_API_KEY</var>' https://api.newrelic.com/graphql
+products: [{id: 9200, name: \"Traces\", unitsOfMeasure: [{unit: SPANS_IN_MILLIONS, quantity: 40}]}]) {enqueued errors {message path } } }"}' -H'Content-type: application/json' -H'Api-key: <var>YOUR_USER_KEY</var>' https://api.newrelic.com/graphql
 ```
 
 ## Product subscriptions for an account [#subscriptions]
 
-The Provisioning API does not allow you to upgrade or downgrade individual product subscriptions for an account. Instead, the API requires you to replace ([add](/docs/new-relic-partnerships/partnerships/partner-api/partnership-api-subscription-object#example-create)) the configuration for all product subscriptions for the account.
+NerdGraph doesn't allow you to upgrade or downgrade individual product subscriptions for an account. Instead, the API requires you to replace ([add](/docs/new-relic-partnerships/partnerships/partner-api/partnership-api-subscription-object#example-create)) the configuration for all product subscriptions for the account.
 
 <Callout variant="tip">
-  If any product configurations are not included, the Provisioning API automatically provisions the account with a free product (when available).
+  If any product configurations are not included, NerdGraph automatically provisions the account with a free product (when available).
```

Comment:
you may want to ask ui-write team how to handle wording on line 33. we don't refer to them as products anymore but i'm not sure that "configurations for New Relic capabilities" is the right way to say this

Reply:
in this case, because this only is relating to the original pricing model, that model still has those as products so think it's fine